evalml/tests/model_understanding_tests/prediction_explanations_tests/test_algorithms.py

```diff
@@ -144,6 +140,29 @@ def test_shap(estimator, problem_type, n_points_to_explain, X_y_binary, X_y_mult
                    shap_values.values()), "A SHAP value must be computed for every data point to explain!"
 
 
+@patch('evalml.model_understanding.prediction_explanations._algorithms.logger')
+@patch('shap.TreeExplainer')
+def test_compute_shap_values_catches_shap_tree_warnings(mock_tree_explainer, mock_debug, X_y_binary, caplog):
```

Comment:
is this new functionality from `0.36.0` and this test is just making sure we catch it?

Reply:
Thanks for asking! I meant to post a comment but forgot. Yea, for some reason, warnings are not being shown when I upgraded to 0.36. Not sure what the root cause is, but rather than deleting the code in question or just merging an uncovered line, I decided to add a test for it.

Reply:
good call!